.travis.yml

```diff
@@ -91,6 +91,29 @@ jobs:
         FILE_ENV="./ci/test/00_setup_env_arm.sh"
         QEMU_USER_CMD=""
 
+    - stage: test
+      name: 's390x native BE [GOAL: install] [bionic] [no depends, no GUI]'
+      arch: s390x
+      dist: bionic
+      cache:
+        directories:
+          - $TRAVIS_BUILD_DIR/ci/scratch/.ccache
+          - $TRAVIS_BUILD_DIR/releases/$HOST
```

Comment:
Why is the overwrite needed?

Reply:
To get rid of smth like this:
```
adding /usr/local/Homebrew to cache
creating directory /usr/local/Homebrew
mkdir: cannot create directory ‘/usr/local/Homebrew’: Permission denied
/usr/local/Homebrew is not yet cached
/usr/local/Homebrew: No such file or directory
/usr/local/Homebrew: unknown file type
```

Reply:
In that case, I'd rather move the overwrite to macos, since that is causing the trouble

Reply:
Done.